TrainingOOP-PHP001/Point.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace TrainingOOPPHP001;
+
+class Point{
+    public $x;
+    public $y;
+    function __construct($x,$y){
```

Comment:
Chú ý có space trước `{`
```suggestion
    function __construct($x,$y) {
```